### PR TITLE
[IMP] use PyPDF2 if possible

### PR DESCRIPTION
**Pull request by hbrunn**
pypdf is very slow for merging pages, with PyPDF2, we get something between 20-80% speedup according to my tests

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Now we have broken tests. Can you please check?

**Review by alexis-via — APPROVED** [review]
Great !

**Comment by lasley** [comment]
> [ADD] can we force some package version on travis?

@hbrunn - You're looking for [`WKHTMLTOPDF_VERSION`](https://github.com/OCA/maintainer-quality-tools/blob/175a25ef71dcae3baa8168852a01ea7831478827/sample_files/.travis.yml#L97) 😉

File: report_qweb_pdf_watermark/models/report.py [hunk 1]
```diff
@@ -43,10 +58,12 @@ def get_pdf(
         pdf = PdfFileWriter()
         pdf_watermark = None
         try:
+            print 'read watermark'
```
Comment (elicoidal) [inline report_qweb_pdf_watermark/models/report.py]: Shouldnot we remove all print statements?

**Comment by hbrunn** [comment]
@elicoidal sure, those were just for debugging why the build wouldn't work on travis
@lasley thanks, I took the liberty to clean that up a bit: https://github.com/OCA/maintainer-quality-tools/pull/479

and yay, it's green, can we merge?

**Comment by pedrobaeza** [comment]
Please change module version number

**Comment by hbrunn** [comment]
done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**